src/components/Explore/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { ListButton, ListContainer, DropdownWrapper } from './Dropdown.style';
+import MenuButton from './MenuButton';
+import { ClickAwayListener } from '@material-ui/core';
+
+const Dropdown: React.FC<{
+  menuLabel: string;
+  itemLabels: string[];
+  onSelect: (itemIndex: number) => void;
+  defaultSelectionLabel: string;
+  maxWidth: number;
+}> = ({ menuLabel, itemLabels, onSelect, defaultSelectionLabel, maxWidth }) => {
+  const [open, setOpen] = useState(false);
+  const [buttonLabel, setButtonLabel] = useState(defaultSelectionLabel);
+
+  const handleClick = (event: any) => {
```

Comment:
Consider updating the `any` with the correct type (or just not add the event, since is not being used)